src/validation.cpp

```diff
@@ -3392,14 +3391,15 @@ std::vector<unsigned char> GenerateCoinbaseCommitment(CBlock& block, const CBloc
 }
 
 //! Returns last CBlockIndex* that is a checkpoint
-static CBlockIndex* GetLastCheckpoint(const CCheckpointData& data) EXCLUSIVE_LOCKS_REQUIRED(cs_main)
+CBlockIndex* BlockManager::GetLastCheckpoint(const CCheckpointData& data)
```

Comment:
626b709edfe1bd71e6d485c096ff9a52f698a94c nit: move help text to header file

Reply:
Will do on next push!